3.9-minimal/test/run

```diff
@@ -6,13 +6,20 @@
 # IMAGE_NAME specifies a name of the candidate image used for testing.
 # The image has to be available before this script is executed.
 #
-declare -a WEB_APPS=({gunicorn-config-different-port,gunicorn-different-port,django-different-port,standalone,setup,setup-requirements,django,numpy,app-home,locale,pipenv,pipenv-and-micropipenv-should-fail,pin-pipenv-version,app-module,micropipenv}-test-app)
```

Comment:
Why did `pin-pipenv-version` test disappear?

Reply:
It's still there but just for full-size container images. Minimal has a new test - pyuwsgi-pipenv.